docs/_posts/Damla-Gurbaz/2022-06-26-icdo_snomed_mapper_en_3_0.md

```diff
@@ -0,0 +1,132 @@
+---
+layout: model
+title: Mapping ICDO Codes with Their Corresponding SNOMED Codes
+author: John Snow Labs
+name: icdo_snomed_mapper
+date: 2022-06-26
+tags: [icdo, snomed, chunk_mapper, clinical, licensed, en]
+task: Chunk Mapping
+language: en
+edition: Spark NLP for Healthcare 3.5.3
+spark_version: 3.0
+supported: true
+article_header:
+  type: cover
+use_language_switcher: "Python-Scala-Java"
+---
+
+## Description
+
+This pretrained model maps ICDO codes to corresponding SNOMED codes.
+
+## Predicted Entities
+
+
+
+{:.btn-box}
+<button class="button button-orange" disabled>Live Demo</button>
+[Open in Colab](https://colab.research.google.com/github/JohnSnowLabs/spark-nlp-workshop/blob/master/tutorials/Certification_Trainings/Healthcare/26.Chunk_Mapping.ipynb){:.button.button-orange.button-orange-trans.co.button-icon}
+[Download](https://s3.amazonaws.com/auxdata.johnsnowlabs.com/clinical/models/icdo_snomed_mapper_en_3.5.3_3.0_1656274513770.zip){:.button.button-orange.button-orange-trans.arr.button-icon}
+
+## How to use
+
+
+
+<div class="tabs-box" markdown="1">
+{% include programmingLanguageSelectScalaPythonNLU.html %}
+```python
+documentAssembler = DocumentAssembler()\
+      .setInputCol("text")\
+      .setOutputCol("ner_chunk")
+
+sbert_embedder = BertSentenceEmbeddings\
+     .pretrained("sbiobert_base_cased_mli", "en", "clinical/models")\
+     .setInputCols(["ner_chunk"])\
+     .setOutputCol("sbert_embeddings")
+ 
+icdo_resolver = SentenceEntityResolverModel\
+     .pretrained("sbiobertresolve_icdo", "en", "clinical/models")\
```

Comment:
Why don't we use `sbiobertresolve_icdo_augmented` model here? Can you use this model and check the results if they are the same please? And please try to use the latest version of these models in sample codes from now.